src/adr/0003-distributed-workqueue.md

```diff
@@ -0,0 +1,26 @@
+# 3. Distributed WorkQueue
+
+Date: 2019-10-02
+
+## Status
+
+Accepted (lazy consensus)
+
+## Context
+
+By switching the task manager to a distributed implementation, we need to be able to run a `Task` on any node of the cluster.
+
+## Decision
+  For the time being we will keep the sequential execution property of the task manager.
+  This is an intermediate milestone forward the final implementation which will drop this property.
+
+ * Use a queue rabbitmq as a workqueue where only the `Created` events are pushed into.
+   This queue will be exclusive and with a `prefetch = 1`.
```

Comment:
This is the technical consequence of the decision. I would split the decisions in two parts:
1. the decision in a non-technical wording "The queue will be exclusive and events will be consumed serially"
2. the decision in technical details "prefetch = 1"